### ISSUE TITLE
[model]실험 내용이력관리

### ISSUE THREAD

**Issue by crystal0112** [sktaiflow/message-generator#11]
[2021/02/09]
1. new_tokenizer(SKT상품, 뉴스내 상품/카테고리 명 추가)기반의 bart모델 학습(loss:  cross_entropy_loss)
데이터: MMS데이터
train_file: /home/x1112373/message-generator/data/cleansed_version/summarization/MMS_train.tsv
test_file: /home/x1112373/message-generator/data/cleansed_version/summarization/MMS_test.tsv
batch_size:2
loss:기본 loss(cross_entropy)
tokenizer: https://github.com/sktaiflow/message-generator/blob/master/tokenizers/emji_tokenizer/model_3991added.json
fine-tuning하지 않고, bart 모델 자체에 대한 학습 진행
학습과정
![image](https://user-images.githubusercontent.com/57606640/107363273-3e659380-6b1d-11eb-9a37-f0efc89d8788.png)

best 에폭 결과: 
![image](https://user-images.githubusercontent.com/57606640/107364960-7e2d7a80-6b1f-11eb-8047-f78449c85f1c.png)

Rouge-Scrore: TBD

**Comment by yyb0330** [comment]
@crystal0112 문자로만 학습한거죠? 그런데 `뉴스 원문 테스트`는 참고로 한거에요?

**Comment by crystal0112** [comment]
> @crystal0112 문자로만 학습한거죠? 그런데 `뉴스 원문 테스트`는 참고로 한거에요?

네네 문자로만 학습했습니다! 뉴스원문 테스트는 참고만 했어요!

**Comment by yyb0330** [comment]
Task(TOS, T deal 등)별 우선순위를 다음 주에 알려줄텐데 지금하고 있는 MMS(TOS)와 최근에 추가한 T deal 우선순위가 높으니 이 두 개를 중점적으로 합시다. 특히, T deal은 결과가 괜찮으면 사업 조직에서 테스트 할 수 있다고 하니 T deal 내용이 잘 생성되는지 봐주세요.

> > @crystal0112 문자로만 학습한거죠? 그런데 `뉴스 원문 테스트`는 참고로 한거에요?
> 
> 네네 문자로만 학습했습니다! 뉴스원문 테스트는 참고만 했어요!

**Comment by gowun** [comment]
@crystal0112 https://github.com/sktaiflow/message-generator/blob/master/models/README.md 여기에 이 결과 특히, 실험내용 표에 추가해 주시면 될거 같습니다!! 
그리고 모델 평가를 위한 메트릭 개발은 제가 연휴 때 해서 코드 올릴게요. test file만 넣으면 자동으로 다 계산되게요!

**Comment by crystal0112** [comment]
> Task(TOS, T deal 등)별 우선순위를 다음 주에 알려줄텐데 지금하고 있는 MMS(TOS)와 최근에 추가한 T deal 우선순위가 높으니 이 두 개를 중점적으로 합시다. 특히, T deal은 결과가 괜찮으면 사업 조직에서 테스트 할 수 있다고 하니 T deal 내용이 잘 생성되는지 봐주세요.
> 
> > > @crystal0112 문자로만 학습한거죠? 그런데 `뉴스 원문 테스트`는 참고로 한거에요?
> > 
> > 
> > 네네 문자로만 학습했습니다! 뉴스원문 테스트는 참고만 했어요!

넵Tdeal데이터 받아서 실험해보겠습니다~

**Comment by crystal0112** [comment]
> @crystal0112 https://github.com/sktaiflow/message-generator/blob/master/models/README.md 여기에 이 결과 특히, 실험내용 표에 추가해 주시면 될거 같습니다!!
> 그리고 모델 평가를 위한 메트릭 개발은 제가 연휴 때 해서 코드 올릴게요. test file만 넣으면 자동으로 다 계산되게요!

네네 좋은 것 같아요!!! https://github.com/seujung/KoBART-summarization/blob/main/rouge_metric.py
요기 매트릭 소스가 있는데 Mecab 설치가 좀 복잡하더라고요. 일단 필요한 파일 + 코드 설정 필요한 부분 제가 적어 둔게 있는데 그거 푸시할게요!
